src/main/java/org/kohsuke/github/AbuseLimitHandler.java

```diff
@@ -89,7 +89,7 @@ public void onError(IOException e, HttpURLConnection uc) throws IOException {
         private long parseWaitTime(HttpURLConnection uc) {
             String v = uc.getHeaderField("Retry-After");
             if (v == null)
-                return 60 * 1000; // can't tell, return 1 min
+                return 61 * 1000; // can't tell, return 1 min plus 1 second for luck
```

Comment:
Is there a functional reason for this?

Reply:
The [GitHub docs](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#exceeding-the-rate-limit) say "Otherwise, wait for at least one minute before retrying." I never know if the check is `< 60s` or `<=60s`, but it seems like waiting something that is unambiguously more than a minute is safest.